src/test/java/com/wikia/webdriver/elements/common/Navigate.java

```diff
@@ -44,4 +44,18 @@ public Navigate toPage(String pageName, String reference) {
 
     return this;
   }
+
+  public Navigate toPage_1(String pageName) {
+    try {
+      URL url = new URL(driver.getCurrentUrl());
+      String host = url.getHost();
+//      String cacheBuster = "?cb=" + DateTime.now().getMillis();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)